ios/OX Coi/Application/AppDelegate.swift

```diff
@@ -5,11 +5,38 @@ import UIKit
 @objc
 class AppDelegate: FlutterAppDelegate {
 
+    private let INTENT_CHANNEL_NAME = "oxcoi.intent"
+    var startString: String?
+    var sharedData: [String: String] = [:]
+
     override func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
         UIApplication.setupLogging()
-        GeneratedPluginRegistrant.register(with: self)
 
+        GeneratedPluginRegistrant.register(with: self)
         return super.application(application, didFinishLaunchingWithOptions: launchOptions)
     }
 
+    override func application(_ app: UIApplication, open url: URL, options: [UIApplication.OpenURLOptionsKey: Any] = [:]) -> Bool {
+        startString = url.absoluteString
+        setupSharingMethodChannel()
+        return true
+    }
+
+    private func setupSharingMethodChannel() {
+        guard let controller = window.rootViewController as? FlutterViewController else {
+            return
+        }
+        let methodChannel = FlutterMethodChannel(name: INTENT_CHANNEL_NAME, binaryMessenger: controller.binaryMessenger)
+        methodChannel.setMethodCallHandler {(call: FlutterMethodCall, result: FlutterResult) -> Void in
+            if call.method.contains("getInitialLink") {
```

Comment:
Today I discussed it with @Boehrsi already. Instead of using magic strings spreaded all over the code, like this one, we should separate all non DCC related, app specific Flutter method names in a separate struct, as we did it with the [DCC related ones](https://github.com/open-xchange/flutter-deltachat-core/blob/develop/ios/Classes/Helper/Method.swift).

So I would suggest to create a file `Method.swift` under `Runner → OX COI → Helper` with:
```
struct Method {}

// Category Foo method calls
extension Method {
   struct Foo {
      static let InviteLink = "getInitialLink"
   }
}

// Category Bar method calls
extension Method {
   struct Bar {
      static let OpenBetterCallSaul = "Better Call Saul"
   }
}
```
to get structured/categorised method names at a central place, just like we do it in the plugin already.

So your code above should then look like:
```
if call.method.contains(Method.Foo.InviteLink) {
```

Reply:
Done